README.md

```diff
@@ -56,21 +60,93 @@ Alternatively, you can download the [release on GitHub](https://github.com/Adyen
+  const { Client } = require('@adyen/api-library');
+  const client = new Client({username: "YOUR_USERNAME", password: "YOUR_PASSWORD", environment: "TEST"});
 ``` 
+
+### Consuming Services
+Every API the library supports is represented by a service object. The name of the service matching the corresponding API is listed in the [Integrations](#integration) section of this document.
+```javascript
+  const { Client, CheckoutAPI	} = require('@adyen/api-library');
+  const client = new Client({apiKey: "YOUR_API_KEY", environment: "TEST"});
+
+  const paymentRequest = {
+    amount: {
+      currency: "USD",
+      value: 1000
+    },
+    reference: "Your order number",
+    paymentMethod: {
+      type: "scheme",
+      encryptedCardNumber: "test_4111111111111111",
+      encryptedExpiryMonth: "test_03",
+      encryptedExpiryYear: "test_2030",
+      encryptedSecurityCode: "test_737"
+    },
+    shopperReference: "YOUR_UNIQUE_SHOPPER_ID_IOfW3k9G2PvXFu2j",
+    storePaymentMethod: true,
+    shopperInteraction: "Ecommerce",
+    recurringProcessingModel: "CardOnFile",
+    returnUrl: "https://your-company.com/...",
+    merchantAccount: "YOUR_MERCHANT_ACCOUNT"
+  };
+  const checkoutAPI = new CheckoutAPI(client);
+  checkoutAPI.payments(paymentRequest)
+    .then( paymentResponse => console.log(paymentResponse.pspReference))
+    .catch(error => console.log(error));
+```
+
+Alternatively you can make use of the Types included in this library using Typescript, and/or use the async syntax:
+```typescript
+  const { Client, CheckoutAPI } = require('@adyen/api-library');
+  const client = new Client({apiKey: "YOUR_API_KEY", environment: "TEST"});
+
+  import { Types } from '@adyen/api-library';
+
+  const doPaymentsRequest = async ()  => {
+    const paymentRequest : Types.checkout.PaymentRequest = {
+      amount: {
+        currency: "USD",
+        value: 1000
+      },
+      reference: "Your order number",
+      paymentMethod: {
+        type: Types.checkout.CardDetails.TypeEnum.Scheme,
+        encryptedCardNumber: "test_4111111111111111",
+        encryptedExpiryMonth: "test_03",
+        encryptedExpiryYear: "test_2030",
+        encryptedSecurityCode: "test_737"
+      },
+      shopperReference: "YOUR_UNIQUE_SHOPPER_ID_IOfW3k9G2PvXFu2j",
+      storePaymentMethod: true,
+      shopperInteraction: Types.checkout.PaymentRequest.ShopperInteractionEnum.Ecommerce,
+      recurringProcessingModel: Types.checkout.PaymentRequest.RecurringProcessingModelEnum.CardOnFile,
+      returnUrl: "https://your-company.com/...",
+      merchantAccount: "YOUR_MERCHANT_ACCOUNT"
+    };
+    const checkoutAPI = new CheckoutAPI(client);
+    const paymentResponse : Types.checkout.PaymentResponse = await checkoutAPI.payments(paymentRequest);
+    console.log(paymentResponse.pspReference);
+  }
+
+  doPaymentsRequest();
+```
  
 ### Custom HTTP Client Configuration
```

Comment:
Is this section maybe superfluous? It looks like a way to call directly the endpoint (passing the API-KEY, etc..) that is outside the scope of the library.

Reply:
It's a way to use another HTTP client from the one that comes with the library. Not sure if and why a merchant would use this as opposed to the default ones, but it is a feature we support for all libraries.